app/views/users/_profile_header.html.erb

```diff
@@ -18,13 +18,13 @@
     </div>
     <div class="profile-details">
       <h1 style="color:<%= HexComparer.new([user_colors(@user)[:bg], user_colors(@user)[:text]]).brightness(0.78) %>">
-        <span itemprop="name"><%= @user.name %></span>
+        <span itemprop="name"><%= sanitize @user.name %></span>
         <span class="user-profile-follow-button-wrapper">
           <button id="user-follow-butt" class="cta follow-action-button user-profile-follow-button" style="color:<%= user_colors(@user)[:text] %>;background-color:<%= user_colors(@user)[:bg] %>" data-info='{"id":<%= @user.id %>,"className":"<%= @user.class.name %>"}'>&nbsp;</button>
         </span>
       </h1>
       <p class="profile-description" itemprop="description">
-        <%= @user.summary.presence || ["404 bio not found"].sample %>
+        <%= sanitize @user.summary.presence || ["404 bio not found"].sample %>
```

Comment:
This can be probably rewritten as `sanitize(@user.summary.presence || "404 bio not found")` without the sample call since there's only one value in the array